tests/playwright-test/esm.spec.ts

```diff
@@ -45,9 +45,15 @@ test('should support import attributes', async ({ runInlineTest }) => {
     'package.json': JSON.stringify({ type: 'module', foo: 'bar' }),
     'a.test.ts': `
       import config from './package.json' with { type: 'json' };
+      import configFooFromUtils from './utils.js'
       console.log('imported value (test): ' + config.foo);
       import { test, expect } from '@playwright/test';
+      expect(configFooFromUtils.foo).toBe('bar');
       test('pass', async () => {});
+    `,
+    'utils.js': `
+      import config from './package.json' with { type: 'json' };
```

Comment:
```diff
- import config from './package.json' with { type: 'json' };
+ import config from './package.json' assert { type: 'json' };
```
shouldn't be?

Reply:
`assert` is the legacy syntax and `with` the newer syntax. Without the plugin Babel is not able to understand `with` - this is what we are fixing here.